api/schemas/report.json

```diff
@@ -588,17 +580,7 @@
                 },
 
                 "desired_value": {
-                    "description": "the value specified in the manifest. Absent in reports of `inspect` kind.",
-                    "oneOf": [
-                        { "type": "string" },
-                        { "type": "array" },
-                        { "type": "object" },
-                        { "type": "null" }
-                    ]
-                },
-
-                "historical_value": {
-                    "description": "The audited value from a previous run of Puppet, if known. Absent in reports of `inspect` kind.",
```

Comment:
I verified the puppetdb report processor doesn't make use of `historical_value`: https://github.com/puppetlabs/puppetdb/blob/master/puppet/lib/puppet/reports/puppetdb.rb